### PR TITLE
New segment test case

### PR DESCRIPTION
**Pull request by escopecz**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [n]
| New feature/enhancement? (use the a.x branch)      | [n]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [y] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

This test was written by @lukassykora as a "Innovation Day" task and tests some segment filters and edit page.

#### Steps to test this PR:

1. No need to test as no production code was changed. Just an automated test was added.

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11404"></a>

### CODE REVIEW

**Comment by escopecz** [comment]
I'm adding the "bug" label so we can add it to the 4.4.2 release. There is no reason to wait for a minor release with this as stability which this new test is checking is always a good thing.

@kuzmany I'm adding you as a reviewer. I don't think it needs more than 1 approval from a developer to merge this.

**Review by kuzmany — APPROVED** [review]
👍

**Pull request merged**
